eng/pipelines/templates/variables/globals.yml

```diff
@@ -14,7 +14,6 @@ variables:
   DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
   DOTNET_CLI_TELEMETRY_OPTOUT: 1
   DOTNET_NOLOGO: 1
-  Package.EnableSBOMSigning: true
   Codeql.SkipTaskAutoInjection: true
```

Comment:
I assume we can also clean out some of these other variables as well.

Reply:
Yeah we will have some follow-on work to remove more scanning redundancies.